packages/cli/src/tools/config/types.flow.js

```diff
@@ -79,9 +79,9 @@ export type ConfigT = {|
 
   // Object that contains configuration for a project (null, when platform not available)
   project: {
-    android: ?ProjectConfigAndroidT,
-    ios: ?ProjectConfigIOST,
-    [key: string]: any,
+    android?: ?ProjectConfigAndroidT,
+    ios?: ?ProjectConfigIOST,
```

Comment:
Shouldn't it be this now?
```jsx
android?: ProjectConfigAndroidT,
ios?: ProjectConfigIOST,
```

Reply:
`android` and `ios` can be missing (no `react-native` in package.json), still they can be `null` when there's `react-native` in a project but no Android or iOS files (say I am working on Android only project).